README.md

```diff
@@ -88,9 +88,9 @@ If you run a business using the project as a revenue-generating product, it make
 
 Nuxt has also been known in the past as "Nuxt.js" and "NuxtJS".
 
-"NuxtJS" is a [registered brand](https://relationshipscience.com/organization/nuxtjs-288590683) by [NuxtLabs](https://nuxtlabs.com), the company run by Nuxt's founders.
+"NuxtJS" is a [registered brand](https://relationshipscience.com/organization/nuxtjs-288590683). It was originally registered by [NuxtLabs](https://nuxtlabs.com), the company created by Nuxt's founders. The Nuxt logo is also a registered trademark. These trademarks are now retained by [Vercel](https://vercel.com/).
```

Comment:
@danielroe 

```
The Nuxt logo is also a registered trademark.
```

Is this accurate? According to the USPTO [status](https://tsdr.uspto.gov/#caseNumber=79394592&caseSearchType=US_APPLICATION&caseType=DEFAULT&searchType=statusSearch:~:text=to%20an%20examiner.-,Status%3A,-A%20final%20Office) the trademark has been refused?

I was trying to look up what fair use entails, is it still okay for me to use the Nuxt trademark e.g. on thumbnails of YouTube videos?

Reply:
trademarks are often granted individually per-jurisdiction

you can see some of the trademarks for the nuxt logo here:

https://branddb.wipo.int/en/similarname/results?sort=score%20desc&start=0&rows=30&asStructure=%7B%22_id%22:%220f72%22,%22boolean%22:%22AND%22,%22bricks%22:%5B%7B%22_id%22:%220f73%22,%22key%22:%22brandName%22,%22value%22:%22nuxt%22,%22strategy%22:%22Simple%22%7D%5D%7D&_=1752309250620&fg=_void_

I can't see a problem at all with your use of it - the logo exists to be used to refer to Nuxt.